From ad0070ad331ad76cf8f5ac678736a5a15108253b Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Tue, 13 Sep 2022 17:25:11 +0200
Subject: [PATCH 214/954] fix(fdts): ddr: add phyvref and sequencectrl for
 STM32MP2

Add two advanced parameters to ease configuration, to replace
hard-coded values in driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 19fb26b5a962f0de874cb1b51f1a231205de6318 fix(fdts): ddr: add phyvref and sequencectrl for STM32MP2]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: I4ada7a052d2a728ef5099abd405dfe66af30f936
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/266084
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp25-ddr.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fdts/stm32mp25-ddr.dtsi b/fdts/stm32mp25-ddr.dtsi
index 1e18f35288..2492bfb062 100644
--- a/fdts/stm32mp25-ddr.dtsi
+++ b/fdts/stm32mp25-ddr.dtsi
@@ -185,6 +185,8 @@
 		DDR_UIA_DISABLEUNUSEDADDRLNS
 		DDR_UIA_PHYINITSEQUENCENUM
 		DDR_UIA_ENABLEDFICSPOLARITYFIX
+		DDR_UIA_PHYVREF
+		DDR_UIA_SEQUENCECTRL_0
 	>;
 
 	st,phy-mr = <
-- 
2.53.0

